product_tags/security/product_template_tag.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.model.access" id="product_template_tag_access_user">
+        <field name="name">product.template.tag access user</field>
+        <field name="model_id" ref="model_product_template_tag"/>
+        <field name="group_id" ref="base.group_user"/>
+        <field name="perm_read" eval="1"/>
+        <field name="perm_create" eval="1"/>
+        <field name="perm_write" eval="1"/>
+        <field name="perm_unlink" eval="1"/>
```

Comment:
Any user can create or delete tags ? IMO using a group would be better.

Reply:
My opinion was the same than for the menu item.